try:
    num1 = int(input("Enter your number: "))
    print(num1+3)
except Exception as e:
    print("An error occurred: ", e)

